src/azure-cli-core/azure/cli/core/prompting.py

```diff
@@ -22,28 +23,64 @@ def _verify_is_a_tty():
         raise NoTTYException()
 
 
-def prompt(msg):
+def prompt(msg, help_string=None):
     _verify_is_a_tty()
-    return input(msg)
+    while True:
+        val = input(msg)
+        if val == '?' and help_string is not None:
+            print(help_string)
+            continue
```

Comment:
This pattern of using `?` to show more info on the prompt is new to me and not used anywhere else in the CLI.
Is there precedent for this?
Also @Azure/azure-cli thoughts on this?